kms-message/src/kms_gcp_request.c

```diff
@@ -84,11 +84,12 @@ kms_gcp_request_oauth_new (const char *host,
    req->crypto.sign_rsaes_pkcs1_v1_5 = kms_sign_rsaes_pkcs1_v1_5;
    if (opt->crypto.sign_rsaes_pkcs1_v1_5) {
       req->crypto.sign_rsaes_pkcs1_v1_5 = opt->crypto.sign_rsaes_pkcs1_v1_5;
+      req->crypto.sign_ctx = opt->crypto.sign_ctx;
```

Comment:
This was a bug discovered when testing the callbacks. The signing context was not being copied from the `kms_request_opt` to the `kms_request_t`.